src/Stripe.net/Services/PaymentIntents/PaymentIntentTransferDataOptions.cs

```diff
@@ -1,14 +1,16 @@
 namespace Stripe
 {
+    using System;
     using Newtonsoft.Json;
 
     public class PaymentIntentTransferDataOptions : INestedOptions
     {
-        [JsonProperty("amount")]
-        public long? Amount { get; set; }
-
         // This is only available on creation and not update
         [JsonProperty("destination")]
         public string Destination { get; set; }
+
+        [Obsolete("Use PaymentIntent.ApplicationFeeAmount instead")]
```

Comment:
Sorry, nitpicking, but can you change this to something like `"Use PaymentIntent*Options.ApplicationFeeAmount"`, or maybe `"Use ApplicationFeeAmount on the parent options class"`?

`PaymentIntent.ApplicationFeeAmount` refers to the property on the entity class, not the options class, so it's a bit misleading.

Reply:
yep that's fair. PTAL